advocate_europe/assets/scss/components/_ideas.scss

```diff
@@ -226,6 +226,10 @@
         display: block;
     }
 
+    .fa-socialshare {
+        @include rem(width, 22px);
```

Comment:
This is not OPIN here ;). In AE we agreed on using only the rem function but not the rem mixin.